extract_endpoint/tests/conftest.py

```diff
@@ -53,3 +54,15 @@ def sample_zipfile(sample_filenames: typing.Tuple[str, str],
     file_z.close()
     file.seek(0)
     return file
+
+
+@pytest.fixture
+def sample_zipfile_fixture(tmpdir: py._path.local.LocalPath,
```

Comment:
This is almost the same as the fixture just above (`sample_zipfile`) - you could you just write that fixture to disk here.